www/js/controllers/menu_controller.js

```diff
@@ -0,0 +1,38 @@
+app.controller('MenuCtrl', function($scope, $state, $ionicLoading, $ionicPopup, BarcodeService, UserService, $ionicHistory) {
+
+  // Logout
+
+  $scope.logout = function(){
```

Comment:
Please fix the spacing with curly braces here too and throughout the app. This should only take a minute or so. You can just do a global search in the `www` directory for `function(){`.